src/ElasticSearchIndexer.php

```diff
@@ -214,4 +225,15 @@ private function extendBatch(string $action, string $indexName, IndexDocument $d
 
         return $params;
     }
+
+    private function parseErrors(array $clientResponse): string
+    {
+        $errors = '';
+        
+        foreach ($clientResponse['items'] as $response) {
+            $response = reset($response);
+            $errors .= $response['error']['reason'] . '; ';
```

Comment:
are we using this reason inside FE ,  did we communicate with them about the glue used?